src/modules/metrics/actions.js

```diff
@@ -22,8 +22,8 @@ export function addMetric(item) {
 
 //spaceId must be done before the metric is removed here.
 export function removeMetrics(ids) {
-  if (ids.length === 0) { return }
   return (dispatch, getState) => {
+    if (ids.length === 0) { return }
```

Comment:
When it was above, it wouldn't have been a valid action.  That's what caused the error.